config/crds/resources/apiextensions.k8s.io_v1_customresourcedefinition_bigqueryroutines.bigquery.cnrm.cloud.google.com.yaml

```diff
@@ -6,16 +6,15 @@ metadata:
   creationTimestamp: null
   labels:
     cnrm.cloud.google.com/managed-by-kcc: "true"
-    cnrm.cloud.google.com/stability-level: alpha
     cnrm.cloud.google.com/system: "true"
-    cnrm.cloud.google.com/tf2crd: "true"
```

Comment:
If this is a Alpha tf resource, removing this line will break the presubmit because there is no controller to reconcile the resource.